### ISSUE TITLE
Nightly builds? Chrome or Chromium?

### ISSUE THREAD

**Issue by mattl** [w3c/wptdashboard#388]
Wondering what would be the best source of nightly builds for GNU/Linux, 64-bit? 

- Chromium has nightly builds, but my understanding is that they may have small differences like missing codecs, EME, etc?

- Chrome Canary doesn't exist for this platform, but there is a 'dev' channel.

**Comment by foolip** [comment]
I think we should use Chrome Dev, since that's what we're using on Travis now.

To test vanilla Chromium builds built from the same commits as Chrome would be interesting for comparison with Chrome and other Chromium-based browsers, but I wouldn't say this is a priority, directly comparing results of (say) Chrome and QQ on Windows would mostly do the job.

**Comment by mattl** [comment]
Good idea. @mariestaver what's a good place to make a note of this as a potential future low-effort browser to add to the stack?

**Comment by mariestaver** [comment]
Snagged it for the backlog and applied appropriate labels -- we'll pull this work into a future sprint where adding a nightly browser makes sense (hopefully soon) -- Thanks for working this out, both of you!

**Comment by mariestaver** [comment]
Also @mattl, I'm going to change the title to be more descriptive of the answer that came out of this discussion; feel free to edit further if needed.

**Comment by mattl** [comment]
I'd like to start testing this with the new Ansible servers by not installing a stable Chrome on the machines.

**Comment by mariestaver** [comment]
Moving to backlog as temporarily blocked by parallelization (to avoid creating unnecessary VMs) -- this may change after some further research/discussion; @mattl or @jugglinmike will update if so.  Thanks!

**Comment by jugglinmike** [comment]
We are collecting results for Chrome's "dev" channel as of commit c8952d37985b44ebba85585f5436051cec425f4a . Since this project's focus was recently narrowed to only concern results collection, that commit resolves this issue.  The wpt.fyi website does not currently display this data, but that is no longer a consideration here. See https://github.com/web-platform-tests/wpt.fyi/issues/60 for details on the status of the feature on that website.
